cpp/src/binary/jit/core/binop.cpp

```diff
@@ -69,7 +70,7 @@ namespace binops {
             return error;
         }
         
-    	gdf_size_type num_chars_bitmask = gdf_get_num_chars_bitmask( num_values );
+    	gdf_size_type num_chars_bitmask = gdf_num_bitmask_elements( num_values );
```

Comment:
rename num_chars_bitmask to num_bitmask_elements or something that does not refer to chars